tests/cli/test_suite.py

```diff
@@ -3114,17 +3114,50 @@ def test_suite_new_profile_runs_notebook_no_jupyter(
     assert len(cells_of_interest_dict) == 1
 
     profiler_code_cell: str = """\
-profiler = UserConfigurableProfiler(
-    profile_dataset=validator,
-    excluded_expectations=None,
-    ignored_columns=ignored_columns,
-    not_null_only=False,
-    primary_or_compound_key=False,
-    semantic_types_dict=None,
-    table_expectations_only=False,
-    value_set_threshold="MANY",
+data_assistant_result: DataAssistantResult = context.assistants.onboarding.run(
```

Comment:
This now includes all settings, including settings that are commented out. To be used as a starting point for documentation.